### PR TITLE
testing facility name migration

### PR DESCRIPTION
**Pull request by nickpell**
adds some utils, and adds migrations for row 2: https://docs.google.com/spreadsheets/d/1ywTP_NU0yMghuUdp3LIejssmNvMdKqQ0-6GRDWJJR8I/edit?ts=59c253d3#gid=673694858

@sheelio @orangejenny

### CODE REVIEW

**Comment by sheelio** [comment]
@nickpell 

Looks good to me.

File: custom/enikshay/management/commands/testing_facility_name2.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    case_type = 'test'
+    case_properties_to_update = [
+        'testing_facility_id',
+        'testing_facility_name',
+    ]
+    datamigration_case_property = 'datamigration_testing_facility_name2'
+
+    @staticmethod
+    def get_case_property_updates(test, domain):
+        if (
```
Comment (sheelio) [inline custom/enikshay/management/commands/testing_facility_name2.py]: @nickpell 

Can we add a check on test.result_recorded = yes here?

**Review by sheelio — CHANGES_REQUESTED** [review]
See my note about checking test.result_recorded

**Review by orangejenny — COMMENTED** [review]
Looks fine once Sheel is satisfied.

**Comment by nickpell** [comment]
@sheelio updated

**Review by sheelio — APPROVED** [review]

**Comment by sheelio** [comment]
Looks good

**Comment by sheelio** [comment]
@nickpell 

Good to merge?  I'd like to build on this.

**Pull request merged**